test/TestHelper.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @description - This class handles the users
+ * */
+class TestHelper {
+  /**
+   * @description - This class handles the users
+   * */
+  constructor() {
```

Comment:
These are constants. Why are there in a class?

Reply:
I planned on adding getter and setter methods to do the class in the future especially if I'm refactoring.

Reply:
Let's wait until then

Reply:
For now, export as a js object